chia/wallet/trading/offer.py

```diff
@@ -155,22 +155,23 @@ def get_offered_coins(self) -> Dict[Optional[bytes32], List[Coin]]:
                 inner_solution: Optional[Program] = get_inner_solution(puzzle_driver, parent_solution)
                 assert inner_puzzle is not None and inner_solution is not None
                 conditions: Program = inner_puzzle.run(inner_solution)
+                spend_additions: List[Coin] = []
```

Comment:
This is nitpicky, but I don't like this rename.  It's specifically the additions with the amount matching the condition that creates the OFFER_HASH.